src/style.css

```
@@ -1 +1,74 @@
-/* Styles go here */
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
```

add styles on class, not tag
```suggestion
.header {
```